packages/core/src/indexing/postgres/migrations/mid-schema-verification.ts

```diff
@@ -16,6 +16,18 @@ export const validTableStructure = {
     nullable: false,
     defaultValue: null,
   },
+  stream_content: {
+    type: 'jsonb',
+    maxLength: null,
+    nullable: false,
+    defaultValue: "'{}'::jsonb",
+  },
+  tip: {
+    type: 'character varying',
+    maxLength: 255,
+    nullable: false,
+    defaultValue: null,
```

Comment:
above this field is set as `notNullable()`.  How can it be not nullable and also have a default value of null?

Reply:
This is a very good question that I don't have a concrete answer to... it is what `knex` returns me when I dump the schema. Looking at the table structure itself it is correct though - my assumption is it is just a default value returned to ensure the structure stays the same between tables when dumped.